crypto_sign/rainbowIIIc-classic/clean/blas_u32.c

```diff
@@ -0,0 +1,87 @@
+#include "blas_u32.h"
+#include "gf.h"
+
+void PQCLEAN_RAINBOWIIICCLASSIC_CLEAN_gf256v_predicated_add_u32(uint8_t *accu_b, uint8_t predicate, const uint8_t *a, unsigned int _num_byte) {
+    uint32_t pr_u32 = ((uint32_t)0) - ((uint32_t)predicate);
+    uint8_t pr_u8 = pr_u32 & 0xff;
+
+    unsigned int n_u32 = _num_byte >> 2;
+    uint32_t *b_u32 = (uint32_t *)accu_b;
+    const uint32_t *a_u32 = (const uint32_t *)a;
+    for (unsigned int i = 0; i < n_u32; i++) {
+        b_u32[i] ^= (a_u32[i] & pr_u32);
+    }
+
+    a += (n_u32 << 2);
+    accu_b += (n_u32 << 2);
+    unsigned int rem = _num_byte & 3;
+    for (unsigned int i = 0; i < rem; i++) {
+        accu_b[i] ^= (a[i] & pr_u8);
+    }
+}
+
+void PQCLEAN_RAINBOWIIICCLASSIC_CLEAN_gf256v_add_u32(uint8_t *accu_b, const uint8_t *a, unsigned int _num_byte) {
+    unsigned int n_u32 = _num_byte >> 2;
+    uint32_t *b_u32 = (uint32_t *)accu_b;
```

Comment:
This is unsafe behaviour if `accu_b` does not meet the `uint32_t` alignment requirements.